src/custom/components/AccountDetails/Transaction/CancelationModal.tsx

```diff
@@ -48,7 +49,7 @@ function RequestCancellationModal(props: RequestCancellationModalProps): JSX.Ele
               <p>
                 This means that a solver might already have included the order in a solution even if this cancellation
                 is successful. Read more in the{' '}
-                <a target="_blank" href="/#/faq/trading#can-i-cancel-an-order">
+                <a target="_blank" rel="noreferrer" href={`${Routes.FAQ_TRADING}#can-i-cancel-an-order`}>
```

Comment:
I think we usually create the links using <StyledInternalLink /> <StyledLink /> or <ExternalLink

is not part of this PR, but just curious if we know why this is like this?

Reply:
Yeah would need to change for sure. In this case I had simply changed the URL but should use the component 👍🏼

Reply:
Just tested this one out when the cancellation modal is open:
- using the <Link> component it opens the page below, which is not great.
- using the <ExternalLink> component I tried not to specify the root domain, but the URL gets malformed: `http://localhost:3000/faq/trading#/can-i-cancel-an-order?chain=rinkeby` 

thinking to leave as is for now in this iteration